### ISSUE TITLE
Using another vocoder

### ISSUE THREAD

**Issue by e0xextazy** [CorentinJ/Real-Time-Voice-Cloning#864]
Thanks for the great job! My question is, can I use another vocoder for waveform synthesis, and if so, what features can it have? (For example: should it have the same sample rate as the entire pipeline?)

**Comment by fancat-programer** [comment]
Yes, you can use a different vocoder, I know for a fact that the encoder can be taught to audio of a different sample rate, but I am not sure about the other models

**Comment by e0xextazy** [comment]
> Yes, you can use a different vocoder, I know for a fact that the encoder can be taught to audio of a different sample rate, but I am not sure about the other models

I also understand that the speaker's encoder should not be affected by this. I'm interested in using a synthesizer and vocoder with different sampel rates.

**Comment by fancat-programer** [comment]
In theory it should work

**Comment by blue-fish** [comment]
It is possible to use a different sample rate for the vocoder. The vocoder hparams must be chosen carefully to be compatible with the synthesizer. There is some discussion here: https://github.com/mozilla/TTS/issues/520#issuecomment-700490677 . If you use our defaults for the synthesizer hparams, the mel spectrograms will not contain any frequency content above fmax=7600 Hz. You may wish to change this.
